app/controllers/register_controller.rb

```diff
@@ -52,7 +52,12 @@ def create
     @change = Change.new(register_name: params[:register], payload: payload, user_id: current_user.id)
     @change.save
 
-    RegisterUpdatesMailer.register_update_notification(@change, current_user).deliver_now
+    @change_approvers = current_user.team_members.first.team.users.where.not('team_members.role' => 'basic').reject{ |u| u == current_user }
```

Comment:
Are you certain it doesn't already reject current users because of team_members?